main.js

```diff
@@ -0,0 +1,196 @@
+//1. Maak een 'Zoek'-knop op de pagina en koppel deze aan een functie die de gegevens over België ophaalt en dit in de console logt. Tip: Als de de documentatie bekijkt en op async zoekt, vindt je een voorbeeld van een GET-request.
+
+async function getBelgium() {
+    const response = await axios.get("https://restcountries.eu/rest/v2/alpha/AW"
+    );
+    const countryName = response.data.name;
+    const subRegion = response.data.subregion;
+    const population = response.data.population;
+    console.log(countryName, "is situated in", subRegion, ". It has a population of", population, "people.");
+    const capital = response.data.capital;
+    console.log("The capital is", capital);
+
+    // opdracht 4, gekopieerd naar de volgende opdracht (lijn58)
+
+
+
+
+    // opdracht 7
+    // ophalen
+    const flagUrl = response.data.flag;
+    const flagIMG = document.getElementById("flag-img");
+    flagIMG.setAttribute("src", flagUrl);
+
+    const countryNameLine = document.getElementById("name-header");
+    countryNameLine.textContent = countryName;
+    document.body.appendChild(countryNameLine);
+
+    const situatedText = document.getElementById("text-line-one");
+    situatedText.textContent = countryName + " is situated in " + subRegion + ". It has a population of " + population + " people.";
+    document.body.appendChild(situatedText);
+
+    const capitalText = document.getElementById("text-line-two");
+    capitalText.textContent = "The capital is " + capital;
+    document.body.appendChild(capitalText);
+}
+
+
+// opdracht 8 bij enter de waarde van belgie
+
+const inputField = document.getElementById("text-input");
+inputField.addEventListener("keyup" , async (e) => {
+    if (e.key === 'Enter') {
```

Comment:
Het liefst wil je zo min mogelijk indenting. Dit kun je doen door de inverse te gebruiken van het statement, en zodra dat waar is te returnen:
```
if (e.key !== 'Enter') return;

blablabla hier komt code zonder extra indent
```